staging/07-querying-flink-tables/src/test/java/marketplace/CustomerServiceTest.java

```diff
@@ -0,0 +1,62 @@
+package marketplace;
+
+import org.apache.flink.table.api.TableResult;
+import org.apache.flink.types.Row;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CustomerServiceTest extends FlinkTableAPITest {
```

Comment:
These are written as "Integration" style tests. They talk to the real cluster. That's not ideal. Many of these tests take 30-90 seconds to execute which is incredibly slow.

However, I don't think we currently have a viable unit test strategy. I am open to alternatives that might help speed things up.

The question is, in the absence of a viable unit test strategy, is this suitable for at least doing internal enablement on the technology?